bin/setup_mongo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Start MongoDB without access control."
+mongod --port 27017 &
```

Comment:
This should probably be `mongod --fork --bind_ip 127.0.0.1 --logpath /dev/stdout --config /etc/mongod.conf` to ensure that it runs with only local access, has the same config as the real instance, and when it returns will be ready for access and so we won't need the `while ! nc ...`.  The reason for the `bind_ip` is to prevent external services from getting a connection followed by a rejection (since auth is on and no user created), and instead just prevent connections altogether.

Not sure if a `--port` is necessary, since the default is `27017`.

(Similar discussion on the official image https://github.com/docker-library/mongo/pull/53)